backend/src/v5/models/revisions.js

```diff
@@ -25,13 +28,13 @@ const Revisions = {};
 const excludeVoids = { void: { $ne: true } };
 const excludeIncomplete = { incomplete: { $exists: false } };
 
-const collectionName = (model) => `${model}.history`;
+const collectionName = (modelType, model) => (modelType === modelTypes.DRAWING ? DRAWINGS_HISTORY_COL : `${model}.history`);
```

Comment:
Can we scrap the constants declaration and if it's DRAWING then just do `{modelTypes}.history`

This is probably the approach we'll move towards on containers/fed